packages/flutter_tools/lib/src/commands/update_packages.dart

```diff
@@ -268,7 +258,7 @@ class UpdatePackagesCommand extends FlutterCommand {
           checksumDependencies[data.name] = data.version;
         }
       }
-      final String checksum = _computeChecksum(checksumDependencies.keys, (String name) => checksumDependencies[name]);
+      final String checksum = _computeChecksum(checksumDependencies.keys, (String name) => checksumDependencies[name]!);
```

Comment:
Kind of weird this doesn't just pass in the `checksumDependencies` map, but the null migration looks fine.